### ISSUE TITLE
Trojan detected by Windows Antivirus

### ISSUE THREAD

**Issue by TakkuZakku** [JannikBirn/palworld-host-save-fix#28]
![image](https://github.com/JannikBirn/palworld-host-save-fix/assets/114043888/e0b2a846-5763-4b2e-97ec-a09152acf90b)

**Comment by Esterkxz** [comment]
Yeah, Chrome and Edge browser has denied access downloaded file is consider a treat.
and Firefox isn't. but, file is go away soon by Windows defender.

**Comment by JannikBirn** [comment]
Since this is not an official tool I don't have the ability to mark this as "virus free" for the different anti virus softwares. (At least I don't know how)
If you're having trouble with your anti virus system and you don't want to ignore the executable in your anti virus software you can always download the source code and run the gui.py script to get the same result as running the .exe .
